app/controllers/users/subscriptions_controller.rb

```diff
@@ -0,0 +1,29 @@
+module Users
+  class SubscriptionsController < ApplicationController
```

Comment:
I think we should move this controller from the users scope. In this case, we will be able to write controller this way:

```ruby
class SubscriptionsController < ApplicationController
  expose :subscription, parent: :current_user

  def create
    subscription.save

    respond_with subscription, location: user_path(subscription.followee)
  end

  def destroy
    subscription.destroy

    respond_with subscription, location: user_path(subscription.followee)
  end

  private

  def subscription_params
    params.require(:subscription).permit(:followee_id)
  end
end
```

Subscribe button will look this way
```ruby
= button_to "Subscribe", subscriptions_path, method: :post, params: { subscription: { followee_id: user.id } }, class: "button"
```

And we should change followees_subscriptions relation in the model (see below in `user.rb`)

What do you think about this variant?

Reply:
It is a good practice to pass parameter explicitly if it is possible.
So, `/subscrptions` url with `follower_id` paramter is better than 
`/users/username/subscriptions` url